recipes/hail/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hail" %}
+{% set version = "0.2.34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 9c3a4e5f81ad23b569c1d6dc80bfb070
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python3 >=3.6,<3.7
+  host:
+    - bokeh <0.14
```

Comment:
Looking at the `setup.py`, I think all you need under `host` is `python`, `pip` and `pytest-runner` and possibly `wheel` (but that might be a dependency that already gets pulled in by `pip`).